nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -302,12 +307,39 @@ boolean symbolHasSuppressWarningsAnnotation(Symbol symbol, String suppression) {
     return false;
   }
 
-  static String errMsgForInitializer(Set<Element> uninitFields) {
+  static int GetLineforField(Element uninitField, VisitorState state) {
```

Comment:
nit: `GetLineNumForField` would be even more self-documenting as the method name, and proper camel case.

Reply:
Yes we need to use camel case here (`getLineNumForField`).  Also, I believe this will work for any `Element` coming from source code, not just fields, right?  In which case, we can name the method `getLineNumForElement` instead.

Reply:
Yes, I will change the method name to getLineNumForElement, that make great sense!